### ISSUE TITLE
new difficulty calculation type = combined difficulty

### ISSUE THREAD

**Issue by Mr-Syntax** [SilentChaos512/ScalingHealth#336]
okay what i mean by this is we all ready have weighted average right, which is supposedly the players combined difficulty divided by the amount of players this is pretty reasonable

but it could be really cool if we could choose to simply add the 2 difficulty levels together when players are near eachother so instead of for example player 1 having 50 in difficulty and player 2 having 90 difficulty resulting in a difficulty of 70
the combined difficulty setting would set the difficulty when those two players are near eachother to 140 difficulty

this is sorta like diablo i think, where if a friend joined it said "the forces of diable grow stronger"

anyway super usefull and nice mod thanks for making it

im on forge 36.16.0 with minecraft 1.16.5

**Comment by Mr-Syntax** [comment]
it could ofc also just be that the server calculates the difficulty of all players in game a combines it so it cant be abused

it would be very helpful when playing the game with a small group of friends

**Comment by najlitarvan** [comment]
there is a globat server difficulty

**Comment by Mr-Syntax** [comment]
like native in minecraft? or is that a setting in scaling health i overlooked?

**Comment by najlitarvan** [comment]
with the older config you would look for this 
#DISTANCE_AND_TIME:
	#Mix of WEIGHTED_AVERAGE and DISTANCE_FROM_SPAWN. Difficulty increases with time but also with distance
	#SERVER_WIDE:
	#Difficulty tracked at the server level. Player difficulty is irrelevant.
	areaMode = "WEIGHTED_AVERAGE" if you set it to server wide it should do the trick
that makes the whole server have one difficulty , you can also take only the highest level

**Comment by Mr-Syntax** [comment]
yeah but what im looking for is like this:

when you move further from spawn the diffuculty increases, but then if a friend joins it simply doubles the difficulty at that distance from origin

**Comment by najlitarvan** [comment]
if you are looking for this, it doesn't exist atm

**Comment by Mr-Syntax** [comment]
thats what i was afraid off, but i hope the devs will see this and maybe add it in a future patch

its just because minecraft gets way too easy the more people you are working together this could help mitigate that

**Comment by najlitarvan** [comment]
well there is one way to increase difficulty somewhat more
	#Mob health settings
	[mob.health]
		#This modifier affects the hp scale of mobs spawned by spawners, a normal 100 hp boost, will be of 30 using default value
		spawner = 0.4
		#Determines how difficulty affects mob health.
		#2 types of modes: Multiplier and additive, with 3 different multipliers.
		#MULTI: Mob hp is multiplied, mobs with higher base hp have more increase.
		#MULTI_HALF: Same as MULTI but mobs with more than 20 hp have reduced scaling (bosses, endermen, witches, etc).
		#MULTI_QUARTER: Same as MULTI_HALF but scaling is reduced further for 20hp+ mobs.
		#ADD: Flat increase for all mobs, no matter their base hp.
		modifierMode = "MULTI_HALF" switch to multi

**Comment by Mr-Syntax** [comment]
yeah but then again if your alone its just way too hard, it would just be nice if it was dynamic with the amount of players ingame, ofc this only works on servers were people play together, if they dont do that this setting would be fucked
